tools/release/README.md

```diff
@@ -1,4 +1,57 @@
-# Checkbox release process (Debian packages)
+# Checkbox release process
+
+> **_NOTE:_** Check the [Launchpad Builders status] before triggering a release.
+Sometimes, the builders will be down and this might prevent building the
+packages.
+
+## Bump the checkbox version
+Run the [Bumpversion workflow] (the default increment is **minor**)  
```

Comment:
Missing a column at the end of the sentence, and probably a blank line between L8 and L9.